src/pages/index.js

```diff
@@ -91,53 +75,106 @@ function renderCard(data) {
 const imagePopup = new PopupWithImage("#preview-image-modal");
 imagePopup.setEventListeners();
 
-//FORM POPUP
+//DELETE CARD POPUP
+const deleteCardPopup = new PopupWithConfirmation("#card__delete-modal");
+deleteCardPopup.setEventListeners();
+
+const profileUpdateSubmit = new PopupWithConfirmation("#profile-edit-modal");
+const newCardSubmit = new PopupWithConfirmation("#add-new-card");
+
+function handleDeleteClick(card) {
+  deleteCardPopup.open();
+  deleteCardPopup.setSubmitAction(() => {
+    deleteCardPopup.setSubmitText(true, "Deleting...");
+    api
+      .deleteCard(card.id)
+      .then(() => {
+        deleteCardPopup.close();
+        card.removeCard();
+      })
+      .catch((err) => console.log(err))
+      .finally(() => deleteCardPopup.setSubmitText(false));
+  });
+}
 
+//FORM POPUP
 const openImagePopup = new PopupWithForm(
   "#add-new-card",
   handleImageFormSubmit
 );
-const editProfile = new PopupWithForm(
-  "#profile-edit-modal",
-  handleFormSubmit
+const editProfile = new PopupWithForm("#profile-edit-modal", handleFormSubmit);
+const profilePicEdit = new PopupWithForm(
+  "#profile__image-edit",
+  profilePicEditSubmit
 );
+
 editProfile.setEventListeners();
+profilePicEdit.setEventListeners();
 openImagePopup.setEventListeners();
 
 const editFormValidator = new FormValidator(config, profileModalForm);
+const profilePicValidator = new FormValidator(config, profilePicModal);
 const cardFormValidator = new FormValidator(config, addNewCardModal);
 
 editFormValidator.enableValidation();
+profilePicValidator.enableValidation();
 cardFormValidator.enableValidation();
 
 //FORM SUBMIT
 function handleFormSubmit(data) {
```

Comment:
There are 3 popups in the project with buttons and 2 forms. You need to **name** the elements the way all could understand what they are. And the functions as well

`handleProfileFormSubmit`

Reply:
You haven't corrected my remark https://github.com/MrSurge87/se_project_aroundtheus/pull/1#discussion_r1387592142